tests/test_ipu_wrapper.py

```diff
@@ -0,0 +1,40 @@
+import unittest as ut
+
+import torch
+from torch_geometric.data import Batch, Data
+
+from graphium.ipu.ipu_wrapper import PyGArgsParser
```

Comment:
```suggestion
 # Run this test only if poptorch is available
            try:
                import poptorch
                from graphium.ipu.ipu_wrapper import PyGArgsParser
            except Exception as e:
                warn(f"Skipping this test because poptorch is not available.\n{e}")
                return

```
Try something like this - not quite sure, but we need to make sure the IPU files aren't imported when in the regular env. 

tests/test_ipu_wrapper.py:6: in <module>
    from graphium.ipu.ipu_wrapper import PyGArgsParser
graphium/ipu/ipu_wrapper.py:20: in <module>
    poptorch = import_poptorch()
graphium/ipu/ipu_utils.py:30: in import_poptorch
    raise e
graphium/ipu/ipu_utils.py:25: in import_poptorch
    import poptorch
E   ModuleNotFoundError: No module named 'poptorch'

Reply:
Tabs are a bit all over the apace on my suggestion there.